package/MDAnalysis/core/universe.py

```diff
@@ -170,62 +229,65 @@ class Universe(object):
 
     Parameters
     ----------
-    topology : str, Topology object or stream
-        A CHARMM/XPLOR PSF topology file, PDB file or Gromacs GRO file; used to
-        define the list of atoms. If the file includes bond information,
-        partial charges, atom masses, ... then these data will be available to
-        MDAnalysis. A "structure" file (PSF, PDB or GRO, in the sense of a
-        topology) is always required. Alternatively, an existing
-        :class:`MDAnalysis.core.topology.Topology` instance may also be given.
-    topology_format
+    topology : `~MDAnalysis.core.topology.Topology`, str, stream, `np.ndarray`, None
+        The topology defines the list of atoms. This can be provided 
+        as filenames, streams of filenames, numpy arrays, or None for 
```

Comment:
Mention you can pass a `mda.Topology` object here?  I think the most prominent option (ie first in lists of possibilities) has to be filename as string as this covers 90% of users, then the more involved options